toml/src/main/kotlin/org/rust/toml/CratesIoApi.kt

```diff
@@ -36,21 +34,36 @@ val CrateDescription.dependencyLine: String
 fun searchCrate(key: TomlKey): Collection<CrateDescription> {
     if (isUnitTestMode) return MOCK!!
 
-    val name = CompletionUtil.getOriginalElement(key)?.text ?: ""
+    // Stuff like [dependencies.'crate'] can contain single quotes which we need to remove or otherwise the crates API
```

Comment:
What kind of strings can be inside `''`? This function will not work for strings containing whitespaces because they cannot be passed as `q=$name`

Reply:
Switched to using `Url` instead of `String` to properly handle this and other special chars